### DISCUSSION TITLE
how to set up view point for visitors

### DISCUSSION THREAD

**Discussion by wedkim1025** [mozilla/hubs#4131] [General Questions & Discussion] [answered]
Hi, I am such a newbie here but wanting to learn.
I have a problem with my hub, so I uploaded a building from blender as gifl file, but the problem is it is not letting me go inside of the building even my camera set is in the building.
is there any special setting for that?  
Thanks !

**Comment by j-conrad** [comment, marked as answer]
It sounds like you are missing a 'Spawn Point' and/or a 'Floor Plan'. 
In Spoke, you'll need to add those items from the 'Elements' section. If you need more information about how those work, check out the following:
https://www.youtube.com/playlist?list=PLCxaiaRxTL6_N3qqMDCeH9U6VpmT0rJMj

**Reply by Softvision-GeluHaiduc** [reply]
Marking this as answered, unmark the answer if you need any more information. Thank you!

**Comment by wedkim1025** [comment]
Thank you so much !

On Mon, Apr 12, 2021 at 12:18 PM j-conrad ***@***.***> wrote:

> It sounds like you are missing a 'Spawn Point' and/or a 'Floor Plan'.
> In Spoke, you'll need to add those items from the 'Elements' section. If
> you need more information about how those work, check out the following:
> https://www.youtube.com/playlist?list=PLCxaiaRxTL6_N3qqMDCeH9U6VpmT0rJMj
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mozilla/hubs/discussions/4131#discussioncomment-601712>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANUMJ2NT42LTNZRLY6TBDBTTINBQZANCNFSM42TYGFLA>
> .
>


-- 
www.wednesdaykim.xyz
